example/lib/main.dart

```diff
@@ -23,12 +23,12 @@ class _MyAppState extends State<MyApp> {
 
   // Platform messages are asynchronous, so we initialize in an async method.
   Future<void> initPlatformState() async {
-    await TorusDirect.init(
+    await CustomAuth.init(
         network: TorusNetwork.testnet,
         browserRedirectUri:
             Uri.parse('https://scripts.toruswallet.io/redirect.html'),
         redirectUri: Uri.parse(
-            'torusapp://org.torusresearch.torusdirectandroid/redirect'));
+            'torusapp://org.torusresearch.customauthandroid/redirect'));
```

Comment:
Please use `torus://org.torusresearch.sample/redirect`, this URL is confusing because Flutter is cross-platform